### PR TITLE
expose log.level for promlog for remote_storage_adapter

### PR DESCRIPTION
**Pull request by sipian**
Fixed DCO for PR #3880 

Closes https://github.com/prometheus/prometheus/issues/3716

Signed-off-by: sipian <cs15btech11019@iith.ac.in>

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
:+1:

**Comment by silentpete** [comment]
👍
@juliusv

File: documentation/examples/remote_storage/remote_storage_adapter/main.go [hunk 1]
```diff
@@ -141,6 +141,7 @@ func parseFlags() *config {
 	)
 	flag.StringVar(&cfg.listenAddr, "web.listen-address", ":9201", "Address to listen on for web endpoints.")
 	flag.StringVar(&cfg.telemetryPath, "web.telemetry-path", "/metrics", "Address to listen on for web endpoints.")
+	flag.StringVar(&cfg.logLevel, "log.level", "debug", "The log level(error/warn/info/debug) to be set for promlog. ")
```
Comment (juliusv) [inline documentation/examples/remote_storage/remote_storage_adapter/main.go]: Users won't know what "promlog" is - I'd use the same flag description as in Prometheus itself: https://sourcegraph.com/github.com/prometheus/prometheus@2bd510a63e48ac6bf4971d62199bdb1045c93f1a/-/blob/vendor/github.com/prometheus/common/promlog/flag/flag.go#L26

Comment (juliusv) [reply documentation/examples/remote_storage/remote_storage_adapter/main.go]: (+ extra whitespace at end of flag description)

**Comment by sipian** [comment]
@juliusv

**Comment by juliusv** [comment]
:+1: Thanks. Right now it'll have to wait one more week or so for our feature moratorium to be over.

**Comment by gouthamve** [comment]
Merging this as moratorium is over and the test failure is unrelated.

**Pull request merged**